wikimedia/wikipedia-ios
# Diffs - Edit attempt schema updates

**Phabricator:** https://phabricator.wikimedia.org/T337331

### Notes
* This PR logs edit actions on article descriptions and talk pages to the EditAttemptSchema

### Test Steps
1. Edit an article description, add a talk page topic, reply to a talk page topic, undo or rollback diffs, and submit these actions.
2. Try it  again, but now close the edit views before submitting
3. Verify that the logs are correctly sent with no errors
